management/src/main/java/org/ehcache/management/providers/CacheBinding.java

```diff
@@ -38,7 +38,7 @@ public String getAlias() {
     return alias;
   }
 
-  public Cache<?, ?> getCache() {
+  public Cache getCache() {
```

Comment:
If I leave them here, `EhcacheActionWrapper` won't compile at lines 64, 71 and 79.

I don't know what would be the best way to solve that... It's caused by the reflection invocation of the management calls (Object type).

```
  @SuppressWarnings("unchecked")
  @Exposed
  public Object get(@Named("key") Object key) {
    Object convertedKey = convert(key, managedObject.getCache().getRuntimeConfiguration().getKeyType());
    return managedObject.getCache().get(convertedKey); // 64
  }

  @SuppressWarnings("unchecked")
  @Exposed
  public void remove(@Named("key") Object key) {
    Object convertedKey = convert(key, managedObject.getCache().getRuntimeConfiguration().getKeyType());
    managedObject.getCache().remove(convertedKey); // 71
  }

  @SuppressWarnings("unchecked")
  @Exposed
  public void put(@Named("key") Object key, @Named("value") Object value) {
    Object convertedKey = convert(key, managedObject.getCache().getRuntimeConfiguration().getKeyType());
    Object convertedValue = convert(value, managedObject.getCache().getRuntimeConfiguration().getValueType());
    managedObject.getCache().put(convertedKey, convertedValue); // 74
  }
```